src/profile-logic/marker-schema.js

```diff
@@ -667,3 +667,23 @@ export function markerPayloadMatchesSearch(
 
   return false;
 }
+
+export function computeStringIndexMarkerFieldsByDataType(
+  markerSchemas: MarkerSchema[]
+): Map<string, string[]> {
+  const stringIndexMarkerFieldsByDataType = new Map();
+  stringIndexMarkerFieldsByDataType.set('CompositorScreenshot', ['url']);
```

Comment:
nit: also add a small comment such as `CompositorScreenshot contains a string index too, but doesn't have a schema because it has a dedicated handling in our code`